tox.ini

```diff
@@ -7,6 +7,7 @@ deps =
      test: pytest
 commands =
      test: py.test []
+     test: py.test doc/source/examples.rst
```

Comment:
It should check the source code within the examples.rst file. As tox is not working on my machine I tried to guess how to incorporate something like:

```
py.test doc/source/examples.rst
```

on my machine, which tests all the code chunks in `example.rst`.